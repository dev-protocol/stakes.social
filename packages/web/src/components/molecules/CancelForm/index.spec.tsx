import React from 'react'
import { render } from '@testing-library/react'
import { CancelForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${CancelForm.name}`, () => {
  test('Snapshot', () => {
    const component = render(<CancelForm onClickCancel={() => {}} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
