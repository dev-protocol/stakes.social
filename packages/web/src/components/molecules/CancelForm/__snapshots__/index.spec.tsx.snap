// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="sc-dy1yq4-0"
    >
      <p>
        Cancel Staking
      </p>
      <button
        class="ant-btn ant-btn-primary ant-btn-lg"
        type="button"
      >
        <span>
          Cancel
        </span>
      </button>
    </div>
  </div>
</body>
`;
